src/hermes/commands/postprocess/invenio.py

```diff
@@ -23,8 +23,8 @@ def config_record_id(ctx):
     try:
         conf.deposit.invenio.record_id = deposition['record_id']
         toml.dump(conf, open('hermes.toml', 'w'))
-    except KeyError:
-        raise RuntimeError("No deposit.invenio configuration available to store record id in") from None
+    except KeyError as e:
+        raise RuntimeError("No deposit.invenio configuration available to store record id in") from e
```

Comment:
I wouldn't `raise from` the `KeyError` here. This will lead to the output

`exception was the direct cause of the following exception`

But it's not. The `KeyError` is just the symptom. The cause is the user not configuring the software right. We don't need to track where the exception came from. We already know.

Reply:
```suggestion
    except KeyError:
        raise RuntimeError("No deposit.invenio configuration available to store record id in")
```

Your such an incredible source of wisdom! Thanks for the hint